frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -87,6 +100,99 @@ class OverrideForkPointDialog(
             thisRowComment
           )
       }
+
```

Comment:
The more I look at that, the more I think that a list of commits **unique** to the branch (i.e. what's gonna be displayed in the status for the branch) would be useful, right above or below the radio buttons... WDYT folks?

Reply:
@PawelLipski 👌